.github/workflows/test-inference.yml

```diff
@@ -0,0 +1,34 @@
+name: Test inference
+
+on: 
+  pull_request:    
+  push:
+    branches:
+      - main
+
+jobs:
+  test:  
+    name: "Test inference"  
+    # This action is designed only to run on the Stability research cluster at this time, so many assumptions are made about the envrionment
+    if: github.repository == 'stability-ai/generative-models'
+    runs-on: [self-hosted, slurm, g40]
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Symlink checkpoints"
+        run: ln -s /admin/home-palp/sgm-checkpoints checkpoints

```

Comment:
@palp do you want to commit this path?

Reply:
Yes, we'll revise it soon as we'll set up a dedicated user/account for the runners.

Reply:
Actually, I've changed my mind and am moving this to a shared path now, which we'll point to via a secret - also making the action a bit closer to being reusable elsewhere.